compiler-java/src/com/redhat/ceylon/ceylondoc/CeylonDocTool.java

```diff
@@ -891,7 +891,7 @@ private void copySourceFiles() throws FileNotFoundException, IOException {
                 markup.around("script src='" + getResourceUrl(pkg, "ceylon.js") + "' type='text/javascript'");
                 markup.around("script src='" + getResourceUrl(pkg, "ceylondoc.js") + "' type='text/javascript'"); 
                 markup.close("head");
-                markup.open("body", "pre data-language='ceylon' style='font-family: Inconsolata, Monaco, Courier, monospace'");
+                markup.open("body", "pre", "code data-language='ceylon' style='font-family: Inconsolata, Monaco, Courier, monospace'");
                 // XXX source char encoding
                 BufferedReader input = new BufferedReader(new InputStreamReader(pu.getUnitFile().getInputStream()));
```

Comment:
It looks like this is going to merge conflict with #6951, because that’s just my luck – of all the lines in the entire repository, two unrelated fixes just had to touch these immediately adjacent lines  :D